scalafix-rules/src/main/scala/scalafix/internal/rule/OrganizeImports.scala

```diff
@@ -776,48 +847,6 @@ object OrganizeImports {
     }
   }
 
-  private def prettyPrintImportGroup(group: Seq[Importer]): String =
-    group
-      .map(i => "import " + importerSyntax(i))
-      .mkString("\n")
-
-  /**
-   * HACK: The Scalafix pretty-printer decides to add spaces after open and
-   * before close braces in imports with multiple importees, i.e., `import a.{
-   * b, c }` instead of `import a.{b, c}`. On the other hand, renames are
-   * pretty-printed without the extra spaces, e.g., `import a.{b => c}`. This
-   * behavior is not customizable and makes ordering imports by ASCII order
-   * complicated.
-   *
-   * This function removes the unwanted spaces as a workaround. In cases where
-   * users do want the inserted spaces, Scalafmt should be used after running
-   * the `OrganizeImports` rule.
-   */
-  private def importerSyntax(importer: Importer): String =
-    importer.pos match {
-      case pos: Position.Range =>
-        // Position found, implies that `importer` was directly parsed from the source code. Returns
-        // the original parsed text to preserve the original source level formatting.
-        pos.text
-
-      case Position.None =>
-        // Position not found, implies that `importer` is derived from certain existing import
-        // statement(s). Pretty-prints it.
-        val syntax = importer.syntax
```

Comment:
This was capturing the implicit `Dialect` always in scope (`Dialect.current`) which matches the runtime Scalafix Scala version. This one is not necessarily the Scala version of the sources (as Scalafix never runs in Scala 3 yet, and as we can have `-Xsource:3` in Scala 2.13).

Moving it outside the companion object allows to capture the freshly-added instance `dialect`, inferred from the `Configuration`.